src/utils/bootstrap.js

```diff
@@ -66,3 +69,10 @@ export function teardownWorkers() {
   stopParserWorker();
   stopSearchWorker();
 }
+
+function updatePrefs(state) {
+  const pendingBreakpoints = selectors.getPendingBreakpoints(state);
+  if (prefs.pendingBreakpoints !== pendingBreakpoints) {
+    prefs.pendingBreakpoints = pendingBreakpoints;
+  }
+}
```

Comment:
this is now responsible for updating bp prefs. it can do more things later if we want.